source/pdk/1.0/pdk_testing.md

```diff
@@ -87,7 +88,7 @@ Before you begin, ensure that you have written unit tests for your module, unles
 pdk test unit
 ```
 
-If there are no errors, this returns successfully as `exit code 0`, with no warnings or errors on zero examples.
+If there are no errors, this returns successfully with an exit code of zero, with no warnings or errors on zero examples. [DavidS: I don't understand the second half here. This is dubiously only true iff there are no tests yet.]
```

Comment:
Ohh, yeah, I know what happened here. This is the result if you are testing a freshly generated module with no added code. Is it still correct to say that if there are no errors, it returns successfully with an exit code of zero?